### ISSUE TITLE
ubuntu上landrop无法连接其他设备

### ISSUE THREAD

**Issue by Spectatorz** [LANDrop/LANDrop#61]
ubuntu上打开landrop，其他设备可以搜索到ubuntu端landrop但是无法将文件传输到ubuntu，显示无法连接，而在ubuntu上发送文件到其他设备时，无法搜索到其他设备

**Comment by uwf-fang** [comment]
Yes. Similar problem happened on my desktop. The Ubuntu desktop is not listed on other device until port is allowed. Still cannot send/receive file. Always timeout.
